AAI/AAIConnectProfile.md

```diff
@@ -338,37 +389,35 @@ the Broker.
             [Metadata](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderMetadata)
             that may be reachable by a Claim Clearinghouse.
 
-        4.  Visa Issuer MUST support public-facing
-            /userinfo endpoint. When presented with a valid Visa Access
-            Token, the /userinfo endpoint MUST return a success status
-            and MAY return the current values for GA4GH Claims that were
-            included within the Visa Access Token, however returning
-            GA4GH Claims from the /userinfo endpoint for Visa Access
-            Tokens is OPTIONAL.
+        4.  Visa Issuer MUST support public-facing /userinfo endpoint. When presented with a valid 
```

Comment:
There are still 3 occurences of `/userinfo` instead of "UserInfo".

Reply:
I will change it. We had discussed reverting this particular section to the previous state and adding a note that Visa Access Tokens were deprecated but I took that revert idea too literally.